src/Xrefcheck/CLI.hs

```diff
@@ -69,6 +73,17 @@ addTraversalOptions TraversalConfig{..} (TraversalOptions ignored) =
   , ..
   }
 
+newtype VerifyOptions = VerifyOptions
+  { voIgnoreLocalhost :: Bool
+  }
+
+addVerifyOptions :: VerifyConfig -> VerifyOptions -> VerifyConfig
+addVerifyOptions VerifyConfig{..} VerifyOptions{..} =
```

Comment:
Minor: the reason why `addTraversalOptions` enlists `TraversalOptions`'s fields in its pattern-match explicitly is that, if someone adds one new option to this datatype, the compiler will complain on `addTraversalOptions` function until it is updated respectively.

But `addVerfiyOptions` currently lacks this property and one can forget to update this function when extending `VerifyOptions` datatype, and so things become slightly more difficult to maintain.

Reply:
Wow, interesting. I didn't think about it. Thanks.